source/dlangbot/bugzilla.d

```diff
@@ -79,3 +81,43 @@ Issue[] getDescriptions(R)(R issueRefs)
         .sort!((a, b) => a.id < b.id)
         .release;
 }
+
+shared string bugzillaLogin, bugzillaPassword;
+
+Json apiCall(string method, Json[string] params)
+{
+    import std.net.curl;
+    import vibe.stream.operations : readAllUTF8;
+    import dlangbot.utils : request;
+
+    string[string] urlParams = [
+        "method" : method,
+    ];
+    static if (!is(P == typeof(null)))
```

Comment:
Hmm, not sure what this line is supposed to mean, but to me this looks like `version (none)`.

Reply:
Oops, that's a left-over from an older implementation